iwyu.cc

```diff
@@ -2693,6 +2684,13 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     return visitor_state_->preprocessor_info;
   }
 
+  bool IsProvidedComponent() const {
```

Comment:
Might also be nice to name this `IsProvidedComponentType` (which seems to be how we're referring to these, though my instinct was that `IsProvidedTypeComponent` would be clearer...)